src/activity_actor_definitions.h

```diff
@@ -1363,4 +1363,44 @@ class haircut_activity_actor : public activity_actor
         static std::unique_ptr<activity_actor> deserialize( JsonIn & );
 };
 
+class rummage_pocket_activity_actor : public activity_actor
+{
+    public:
+        enum class action : int {
+            activate,
+            drop,
+            empty,
+            none,
+            read,
+            wear,
+            wield,
+            last
+        };
+    private:
+        using item_locations = drop_locations;
+        item_locations item_loc;
```

Comment:
What is the point of having a list of item locations when you only use the first one?
You also never use the count.

Reply:
This is necessary for when the activity is called for the `drop` case, it passes the whole `drop_location` for the `drop` function used in `rummage_pocket_activity_actor::finish`.

Reply:
I missed that, my bad.